examples/zolertia/z1/Makefile

```diff
@@ -7,8 +7,11 @@ ZOLERTIA_Z1SP=0
 
 CONTIKI_PROJECT = test-phidgets blink test-adxl345 test-tmp102 test-light-ziglet
 CONTIKI_PROJECT += test-battery test-relay-phidget test-tlc59116 test-sht25
-CONTIKI_SOURCEFILES += sht11.c reed-sensor.c
-APPS=serial-shell
+
+CONTIKI_SOURCEFILES += reed-sensor.c sht25.c tlc59116.c light-ziglet.c \
+                       relay-phidget.c sht11.c sht11-sensor.c
+
+MODULES += dev/sht11
```

Comment:
I must apologize; when I posted my comment about `sht11.c`, I had not thought, yet, about the meaning of this `MODULES` line.  It compiles the two "sht11*.c" files; therefore, they do not need to be named in the above `CONTIKI_SOURCEFILES` line.